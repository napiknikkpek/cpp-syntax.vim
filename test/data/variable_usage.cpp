foo::a + foo::b;
foo::a.x;
foo::a->x;
foo<bar>::a;
a * b;
a & b;
