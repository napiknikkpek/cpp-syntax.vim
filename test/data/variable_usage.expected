ttt..x...ttt..x.
ttt..x.x.
ttt..x..x.
ttt.ttt...x.
x...x.
x...x.
